app/controllers/registration_controller.rb

```diff
@@ -11,14 +12,13 @@ def create
       firstname: oauth_data["givenName"], 
       lastname: oauth_data["sn"]
     }
-    user = User.create(create_params.merge(oauth_params))
-    if(!user.valid?)
-      @errors = user.errors
+    @user = User.create(create_params.merge(oauth_params))
+    if(!@user.valid?)
       render :new
     else
```

Comment:
i think this if-else is a bit twisted.

what do you think about handling the positive case in `if` and the negative case in `else`?
`if !condition` should be `unless` but if you rewrite it you can see that it will look like this: `unless condition ... else ... end`